core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -383,33 +383,21 @@ final case class App(id: AppId,
                      extraArgs: List[String]
 ) {
 
-  def getProxyUrls(cloudContext: CloudContext,
-                   workspaceId: Option[WorkspaceId],
-                   proxyUrlBase: String,
-                   apiVersion: String
-  ): Map[ServiceName, URL] =
+  def getProxyUrls(cluster: KubernetesCluster, proxyUrlBase: String): Map[ServiceName, URL] =
```

Comment:
I realized this logic wasn't correct (despite me 👍 ing it in a previous PR) -- for Azure the proxy path should be based on Azure relay, not `/proxy/apps/v2/...`.

This logic now returns the correct proxyPath based on whether the app is GCP or Azure.